### ISSUE TITLE
[Marketplace] If a user is an admin or owner in multiple shops, they should be able to switch permission and shop dashboard contexts

### ISSUE THREAD

**Issue by spencern** [reactioncommerce/reaction#2191]
Users with owner/admin permissions for multiple shops should be able to switch the shop context in which they are viewing the dashboard, catalog and products.

Could be a dropdown to just switch contexts, maybe next to the username dropdown.
Could also be done with expanding and collapsing panels, though that wouldn't transfer to the pdp/grid view as well.

```
===================
BlueStuff (A4F39B)
--------------------------
Accounts
Orders
etc
===================
Red Shirts (12399A)
===================
MyShop3 (987BBC)
===================
```

**Comment by rymorgan** [comment]
**Links for designs below** 

Design annotations in Zeplin :)

Reaction core team link: 
Web: https://zpl.io/1YphHq
Zeplin App: zpl://project?pid=59487558eca97e6140dd4b5b

Public link: 
https://scene.zeplin.io/project/59487558eca97e6140dd4b5b

**Comment by impactmass** [comment]
this seems completed already by @spencern 's #2429?

**Comment by spencern** [comment]
Yeah, this is done-ish. I need to add a way to switch back to the main shop with "shopper" permissions if you're only an admin for a merchant shop

**Comment by aldeed** [comment]
Leaving this open because we lost this selector when moving to new Catalyst UI, and we need to add it back. @rymorgan

**Comment by rymorgan** [comment]
@aldeed We have a plan for shop switcher in the catalyst UI but I didn't think we had official support for this yet?

**Comment by aldeed** [comment]
@rymorgan I think whoever adds the shop switcher would at the same time need to add some "current shop" state tracking and make sure that the UI updates and re-queries when this changes. There shouldn't be a ton of work beyond that because the API already fully supports multi-shop. Possibly we'll need to clean/move/fix some of the shop creation code/UI, too.

**Comment by rymorgan** [comment]
I suppose the shop switcher could be some sort of drop down in the shop settings for now but our long term vision is to is the logo area for the shop switcher.
